.github/CODEOWNERS

```
@@ -1,5 +1,9 @@
 # fdc3-maintainers will be default reviewers for everything in the repo unless a later match takes precedence
 * @finos/fdc3-maintainers
 
+# fdc3-editors will be default reviewers for docs and schemas
+/website/docs/* @finos/fdc3-editors
+/packages/fdc3-context/schemas/context/* @finos/finos-editors
```

Github is highlighting one of the teams (@finos/fdc3-editors)  as unknown, while the other is maybe not the right one?

This does exist however: @finos/fdc3-editors so I suspect its that the editors don't have write permissions. Perhaps we should ask FINOS to change that.

We also need these reviews on fdc3-for-web and birdging schemas.

Hence, how about:
```suggestion
/website/docs/* @finos/fdc3-editors
/packages/fdc3-context/schemas/context/* @finos/fdc3-editors
/packages/fdc3-schema/schemas/* @finos/fdc3-editors 
```